financialaid/models.py

```diff
@@ -10,20 +9,42 @@
 from jsonfield import JSONField
 
 from courses.models import (
-    CourseRun,
     Program,
 )
 
 
-class Tier(models.Model):
+class TimeStampedModel(models.Model):
+    """
+    Base model for create/update timestamps
+    """
+    created_on = models.DateTimeField(auto_now_add=True)  # UTC
+    updated_on = models.DateTimeField(auto_now=True)  # UTC
+
+    def update(self, **kwargs):
```

Comment:
I'm having trouble getting this to work. My code:

    In [24]: t = Tier.objects.create()
    
    In [25]: t.id
    Out[25]: 4
    
    In [26]: t.updated_on
    Out[26]: datetime.datetime(2016, 9, 21, 13, 0, 6, 931005, tzinfo=<UTC>)
    
    In [27]: t.save()
    
    In [28]: t.updated_on
    Out[28]: datetime.datetime(2016, 9, 21, 13, 0, 16, 165478, tzinfo=<UTC>)
    
    In [29]: Tier.objects.filter(id=t.id).update(name='new name')
    Out[29]: 1
    
    In [30]: Tier.objects.get(id=t.id).updated_on
    Out[30]: datetime.datetime(2016, 9, 21, 13, 0, 16, 165478, tzinfo=<UTC>)

How is this function supposed to be used?